cmd/nerdctl/images_test.go

```diff
@@ -116,3 +116,22 @@ LABEL version=0.1`, testutil.CommonImage)
 	base.Cmd("images", "--filter", "reference=busy*:*libc*").AssertOutContains("glibc")
 	base.Cmd("images", "--filter", "reference=busy*:*libc*").AssertOutContains("uclibc")
 }
+
+func TestDanglingImagesFilter(t *testing.T) {
```

Comment:
```suggestion
func TestImagesFilterDangling(t *testing.T) {
```

Similar tests should have a common prefix